arches.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Not intended to be run directly.
+echo "  Linux i386..."
+GOOS=linux GOARCH=386 ./build.sh
+echo "  Linux amd64..."
+GOOS=linux GOARCH=amd64 ./build.sh
+echo "  Linux arm..."
+GOOS=linux GOARCH=arm64 ./build.sh
+echo "  Linux arm64..."
+GOOS=linux GOARCH=arm64 ./build.sh
+echo "  macOS amd64..."
+GOOS=darwin GOARCH=amd64 ./build.sh
+echo "  macOS arm64..."
+GOOS=darwin GOARCH=arm64 ./build.sh
```

Comment:
Because of `brew install moar` I don't think macOS binaries are worth much effort.

If macOS `arm64` should be added, it should be done as a Universal Binary together with the `amd64` one.